sdk/monitor/monitor-query/src/logsClient.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureLogAnalytics } from "./generated/logquery/src/azureLogAnalytics";
+import { TokenCredential } from "@azure/core-auth";
+import { BatchRequest, QueryBatchResponse } from "./generated/logquery/src";
+import { QueryLogsBatchOptions, QueryLogsOptions, QueryLogsResult } from "./models/logsModels";
+import { formatPreferHeader } from "./internal/util";
+
+export interface LogsClientOptions {
```

Comment:
not extending the base client options? (which for v1 is PipelineOptions)

Reply:
I can do that. I didn't intentionally target the old v1 so I might do a regen after this (and still extend the right options).

Reply:
Done. I also added in options for MetricsClient and did the same there.